cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,148 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const user = {
+    username: 'lara_c',
+    password: 'Password!1'
+  };
 
+  const book = {
+    title: 'Speaking JavaScript',
+    author: 'Axel Rauschmayer',
+    publisher: 'O\'Reilly Media',
+    description: 'Like it or not, JavaScript is everywhere these day'
+  };
+
+  const allertMessage = {
+    added: 'Book added to your collection.',
+    deleted: 'Book deleted.'
+  };
+
+  beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  it.only('should allow to login', () => {
```

Comment:
You are using `it.only` which means only this test will run. If you want to run all tests, you should replace it with `it`